Dockerfile

```diff
@@ -16,6 +16,3 @@ RUN apt-get update && apt-get install -y \
 ENV MARU_WORKSPACE /var/maru
 RUN mkdir -p ${MARU_WORKSPACE}
 WORKDIR ${MARU_WORKSPACE}
-COPY . ${MARU_WORKSPACE}
```

Comment:
Thanks for removing this and using a volume mount in the script instead. I think that's much cleaner.